src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,27 +9,329 @@ service: CloudCannon
+
+A good way to ensure that you can use CloudCannon seamlessly with Astro content collections is to [create a CloudCannon schema](#create-a-cloudcannon-schema-for-a-collection) for each Astro content collection. This schema is a template for a new entry, and you can include all the required frontmatter fields to satisfy your Astro content collection's Zod `schema`.
+
+### Create a CloudCannon schema for a collection
+
+To ensure that the data properties of your CloudCannon entries match the Zod validation `schema` defined in your content collection, you can create a [CloudCannon schema](https://cloudcannon.com/documentation/articles/what-is-a-schema/) (a blank template document for creating a new entry). Creating a template schema can ensure that any new documents created in CloudCannon will have the properties required by your content collection and avoid type errors in your project. Your CloudCannon schema can also include default values to start new documents, such as an author name for a single person blog.
+
+The following example will create a CloudCannon schema based on an Astro content collection (`blog`) for blog posts written in Markdown. This schema will be available when [creating a new entry](#creating-a-new-entry) from the CloudCannon "Posts" collection:
+
+<Steps>
+
+1. Create a folder at `.cloudcannon/schemas/` if it does not already exist.
+2. Add an existing blank file in this folder to be used as a default blog post template. The name is unimportant, but the file must have the same file extension as your Astro content collection entries (e.g. `post.md`).
+3. Provide the necessary frontmatter fields required by your content collection's schema. You do not need to provide any values for these, but any content you do include will be automatically included when a new entry is created. These are fields that will be available in the sidebar of your Content Editor.
+
+      The following example schema for a blog post has placeholders for the title, author, and date:
+
+      ```markdown title=".cloudcannon/schemas/post.md"
+      ---
+      title:
+      author:
+      date:
+      ---
+      
+      ```
+4. In your Cloudflare configuration file's `collections_config` property, add the file path to your schema under the CloudCannon collection under the "Posts" collection.
+
+      ```yml title="cloudcannon.config.yml" ins={6-9}
+      collections_config:
+        posts:
+          path: content/blog
+          name: Posts
+          icon: post_add
+          schemas:
+            default:
+              path: .cloudcannon/schemas/post.md
+              name: Blog Post Entry
+      ```
+</Steps>
+
+### Configuring multiple schemas
```

Comment:
While totally correct, I think one important detail about this section is that it's entirely specific to CloudCannon and not mandatory if you want to understand how to use CloudCannon & Astro together.

I guess the question would be if the guide should be a thorough guide on how CloudCannon & Astro works together and can be used together, or should it also cover some advanced CloudCannon patterns.